### ISSUE TITLE
Scores From Prediction Databases?

### ISSUE THREAD

**Issue by jennylsmith** [KechrisLab/multiMiR#28]
Hello, 

Thanks for creating such a comprehensive database of miRNA-mRNA interactions. I was wondering if you could add some documentation regarding the scores for the prediction databases? I read your original manuscript, and I found the line, "All predicted databases have a primary score for target site strength, which is calculated depending on the particular computational method." Does this mean each score is imported from the respective prediction databases? So that a score from a hit in PicTar would not be comparable to a score from miRanda for the same miRNA-mRNA pair? Or does multiMiR calculate its own confidence score? 

Thanks, 

Jenny

**Comment by smahaffey** [comment]
Yes the score is imported from the source and is not comparable between databases.  When you search the search is being limited to the top percentage or number of hits from each source.  For detail on how each score was calculated you would have to look at the documentation from the source.  I hope that is clear.  Sorry for the slow response.

**Comment by duz39** [comment]
Hello -- I have the same question and was unable to clarify interpretation of the prediction scores. I recognize that each database calculates its own prediction score, but many of them calculate multiple scores and it is unclear which of these the multiMiR output includes. Further, I can't find information about the prediction score calculation for some of the databases (e.g. microcosm). It is difficult to filter by score without knowing what prediction scores signify or the suggested cutoff for the different databases. The information that is available in the Ru et al 2014 multiMiR publication also doesn't seem to align with the multiMiR output I produced in 2019 (i.e., for PITA, the publication discusses a prediction score of 0 to 1, but my prediction scores range from -7 to -22. I assume this is actually PITA's delta delta G score, but I'm not sure. Is this information available somewhere? Thanks so much for making this tool publicly available.
